src/main/java/edu/cornell/kfs/coa/document/CuSubAccountMaintainableImpl.java

```diff
@@ -358,10 +358,10 @@ private boolean isIcrSectionDataChanged(List<A21IndirectCostRecoveryAccount> new
     /**
      * After a copy is done set specific fields on {@link SubAccount} to default values
      *
-     * @see org.kuali.rice.kns.maintenance.KualiMaintainableImpl#processAfterCopy(MaintenanceDocument, Map)
+     * @see org.kuali.rice.kns.maintenance.KualiMaintainableImpl#processAfterCopy(org.kuali.rice.kns.document.MaintenanceDocument, Map)
      */
     @Override
-    public void processAfterCopy(org.kuali.rice.kns.document.MaintenanceDocument  document, Map<String, String[]> parameters) {
+    public void processAfterCopy(MaintenanceDocument  document, Map<String, String[]> parameters) {
```

Comment:
The old import was                    org.kuali.rice.krad.maintenance.MaintenanceDocument
the old full qualified object was org.kuali.rice.kns.document.MaintenanceDocument
The new import is                     org.kuali.kfs.kns.document.MaintenanceDocument

Do we need this to be fully qualified to org.kuali.kfs.krad.document.MaintenanceDocument (if it exists)